zap/src/main/java/org/zaproxy/zap/view/MultipleRegexesOptionsPanel.java

```diff
@@ -211,23 +227,23 @@ protected void init() {
 
         @Override
         protected boolean validateFields() {
-            try {
-                Pattern.compile(getRegexTextField().getText(), Pattern.CASE_INSENSITIVE);
-            } catch (PatternSyntaxException e) {
+            String trimmedRegex = getRegexTextField().getText().trim();
+            if (regexes != null && regexes.contains(trimmedRegex)) {
                 JOptionPane.showMessageDialog(
                         this,
-                        MessageFormat.format(TEXT_INVALID_REGEX_DIALOG, e.getLocalizedMessage()),
-                        TITLE_INVALID_REGEX_DIALOG,
+                        TEXT_ALREADY_EXIST_INVALID_REGEX_DIALOG,
+                        TITLE_ALREADY_EXIST_INVALID_REGEX_DIALOG,
                         JOptionPane.INFORMATION_MESSAGE);
                 getRegexTextField().requestFocusInWindow();
                 return false;
             }
+            Pattern.compile(trimmedRegex, Pattern.CASE_INSENSITIVE);
```

Comment:
This lost the catch and error message, was that on purpose?

Reply:
No.